front/app/hooks/useIdeasFilterCounts.ts

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from 'react';
+
+// services
+import {
+  IIdeasFilterCounts,
+  ideasFilterCountsStream,
+  IIdeasFilterCountsQueryParameters,
+} from 'services/ideas';
+
+// utils
+import { omitBy, isNil } from 'lodash-es';
+
+// typings
+import { NilOrError } from 'utils/helperUtils';
+
+export default function useIdeasFilterCounts(
+  props: IIdeasFilterCountsQueryParameters | null
+) {
+  const [ideasFilterCounts, setIdeasFilterCounts] = useState<
+    IIdeasFilterCounts | NilOrError
+  >();
+
+  const propsStr = props === null ? null : JSON.stringify(omitBy(props, isNil));
```

Comment:
```suggestion
    const propsStr = JSON.stringify(omitBy(props, isNil));
```
If we leave out the null type, this can also be simplified.